### ISSUE TITLE
test 2

### ISSUE THREAD

**Issue by syrym-almaty** [iitu2116-7/backend#1]
# check test

- Толеуханова Зарина IT2-2217
- Ким Виктория IT2-2217
- Шадияров Армат IT2-2216

**Comment by syrym-almaty** [comment]
@toleukhanovazarina 
@iitu2116-7/dev-team-1 

give me a feedback

**Comment by syrym-almaty** [comment]
> # toleukhanovazarina @Guacam0lee https://t.me/Guacam0lee
> # arshadiyarov
> # kimchishkav

**Comment by toleukhanovazarina** [comment]
?

**Comment by toleukhanovazarina** [comment]
Can we create separate repo for front?

**Comment by syrym-almaty** [comment]
> Can we create separate repo for front?

Да, вы можете тут прямо создать отдельный репозитории для фронта, у вас вроде есть на это право доступа
